R/compute_risk.R

```diff
@@ -8,15 +8,19 @@
 #' @param h A scalar representing the bandwidth value.
 #' @param matched_Y A vector of outcome variable in the matched set.
 #' @param matched_w A vector of continuous exposure variable in the matched set.
+#' @param matched_cw A vector of counter or weight variable in the matched set.
```

Comment:
the variable names "matched_w" and "matched_cw" are too similar to each other, yet represent very different means. I would suggest just calling it "weights"

Reply:
Should we completely drop `matched_` and go with `Y`, `w`, and `counter_weight`?

Reply:
Yes, that is a great idea!